tasks/main.yml

```diff
@@ -1,5 +1,35 @@
 ---
 
+
+- name: Show Clean Insights measurement collection disclaimer
+  debug:
+    msg: |
+      By continuing with the ongoing installation process,
+      you are giving your consent for the collection of the following
+      Clean Insights measurements:
+        - Installed OpenWISP Version
+        - Enabled OpenWISP modules: A list of the enabled OpenWISP modules
+          along with their respective versions
+        - OS details: Information on the operating system, including its version, kernel version, and platform
```

Comment:
I think it's easier to just link to a page which explains the data we collect, so we have to maintain this info only in one place.